pyannote/audio/core/inference.py

```diff
@@ -408,6 +420,17 @@ def crop(
         >>> inference.crop(file, extended_chunk).crop(chunk_of_interest, returns_data=False)
         """
 
+        if (self.device.type == "cuda") and (
+            torch.backends.cuda.matmul.allow_tf32 or torch.backends.cudnn.allow_tf32
+        ):
+            warnings.warn(
+                "TensorFloat-32 (TF32) is enabled, which might lead to "
+                "numerical instabilities. Please disable it by calling "
```

Comment:
maybe say something about reproducibility of results with different cuda/cudnn versions too ?